apps/fulltextsearch.sh

```diff
@@ -47,13 +49,13 @@ else
         fi
     done
     # Removal Docker image
-    docker_prune_this "$nc_fts"
```

Comment:
we should probably leave this here for cleaning up any potential old installation

Reply:
we should probably also remove the esdata volume in that case

Reply:
Oh! i didn't realized I've removed that. I'll restore it.
IIRC docker system prune -a should remove all images and volumes, right?

Reply:
> system prune -a should remove all images

it removes all dangling images and networks, afaik. volumes need to be purge with docker volume prune but I am not sure if thats the best way to do it. deleting this volume specifically is probably better...

Reply:
yes but as I said i think it is better to just remove the specific volume here...

Reply:
fixed by 61bd632